packages/docs/netlify.toml

```diff
@@ -29,7 +29,7 @@
 # COMMENT: For sharedArrayBuffer, see https://developer.chrome.com/blog/enabling-shared-array-buffer/
 [[headers]]
   # Define which paths this specific [[headers]] block will cover.
-  for = "/live-examples/*"
```

Comment:
@sedghi - this is the fix for the deploy not working.  I'm open to other examples of how to deploy this in terms of a different pattern, but for now I'd like to leave this one as is.